gpt_engineer/ai.py

```diff
@@ -152,3 +227,14 @@ def fallback_model(model: str) -> str:
             "https://openai.com/waitlist/gpt-4-api\n"
         )
         return "gpt-3.5-turbo-16k"
+
+
+def serialize_messages(messages: List[Message]) -> str:
+    return AI.serialize_messages(messages)
+
+
+def cleanup_reponse(response: str) -> str:
+    response = re.sub(
+        "\\n", "\n", response
+    )  # for some reason models sometimes return \n instead of newline?
+    return response
```

Comment:
This will break when there is code that contains strings with \n no?

Would prefer if we let the LLM object handle this

Reply:
Agree & changed